Cue/app/actions/library.js

```diff
@@ -130,12 +131,43 @@ async function addLibrary(uuid: string): PromiseAction {
     }
   }
 
-  let deck = await LibraryApi.fetchDeck(uuid);
+  let addedDeck = await LibraryApi.fetchDeck(uuid);
 
   return {
     type: 'DECK_ADDED_TO_LIBRARY',
-    deck,
+    addedDeck,
   };
 }
 
-module.exports = { loadLibrary, createDeck, deleteDeck, editDeck, recordShareCode, syncDeck, syncLibrary, addLibrary, resolveConflict };
+function clearInaccessibleDecks(): Action {
+  return {
+    type: 'CLEAR_INACCESSIBLE_DECKS'
+  }
+}
+
+function copyDeck(localDeck: Deck) : Action {
+  let cards = JSON.parse(JSON.stringify(localDeck.cards))
```

Comment:
Why the parse(stringify)? Shouldn't localDeck.cards should be a JSON object already?